README.md

```diff
@@ -202,7 +202,9 @@ Manually tested in:
         "dataSource": {
             "url": "...",
             "type": "longPoll",
-            "refresh": 10000
+            "refresh": 10000,
+            "useXDR": true || false // see https://github.com/Raynos/xhr#optionsusexdr for usage
+                                    // if undefined will default to true
```

Comment:
I think the request related options (url and useXDR in this example) need to move into a object of their own if we are using the 'overriding default options' pattern for `xhrOptions`. In this flattened structure the `xhrOptions` would end up including the `type` and `refresh` properties as well once the config object was merged onto the default object (https://github.com/mediasuitenz/mappy/pull/46/files#diff-c385ce3d4304f2e93613b211f96a38b9R14), which aren't valid options to `xhr`.

Namespacing the request related config into a separate object would be much safer I think, similar to my example from yesterday.